### ISSUE TITLE
Compatible with page forms?

### ISSUE THREAD

**Issue by luthfii** [SemanticMediaWiki/SemanticSignup#30]
How is the compatibility with Page form extension? I tried install with page form then i got this error message. 

> Error: This version of SemanticSignup is only compatible with Semantic Forms 2.8 or above. You need to upgrade Semantic Forms first.

**Comment by kghbln** [comment]
I think this extension is already broken with the later releases of Semantic Forms and probably needs some extra adjustments to work with Page Forms.

**Comment by mwjames** [comment]
> I think this extension is already broken with the later releases of Semantic Forms

I'd think before that MW 1.27 `AuthManager` changes make it unlikely to be run on the latest LTS. As for page forms, certainly some changes are necessary.

As a reference, see:
- https://github.com/SemanticMediaWiki/SemanticFormsSelect/pull/38
- https://github.com/SemanticMediaWiki/SemanticFormsSelect/commit/483f3702a7daa9f3b98a6f9b7daa2288b6c8978e

**Comment by kghbln** [comment]
> I'd think before that MW 1.27 AuthManager changes make it unlikely to be run on the latest LTS

Indeed, that's another big issue already recorded here: https://github.com/SemanticMediaWiki/SemanticSignup/issues/29

**Comment by kevindank** [comment]
Subscribing to this thread, it would fantastic to use SSF on my wiki.
